### ISSUE TITLE
ToDos/Notes for template

### ISSUE THREAD

**Issue by gabroko** [IEA-EBC-Annex86/annex#21]
- allow only predefined values for Type of building in META home (see definition)

**Comment by retostauffer** [comment]
@gabroko 

Implemented in version `0.2-13` (current main branch on github)

* Updated XLSX template, "Type of building" colored in dark gray and added data validation (dropdown) to only allow for the values specified in the Definitions table. Leaving it empty is _not_ an option.
* `annex_validate()` checks if this column is filled and throws warnings otherwise.
    * Note: `annex_validate()` does not test if the content of the cells is one of the allowed types, Excel is doing that. This is the same as we use for other variables (e.g., Ventilation type).

Feel free to close this issue if you think that this job is done.

**Comment by gabroko** [comment]
Thanks! But actually it isn't a mandatory field, so no coloring in dark gray and no warning needed  (only a note during the valodate). Just like for the field  Method of vent. rate determination, here we also have predefined values but not mandatory.
Yes, if excel checks, that is certainly enough... No big deal, if it stays a while like that, but for the next change...

**Comment by retostauffer** [comment]
@gabroko META-Home "Ventilation type" was already set "mandatory" and `annex_validate()` will warn the user if the cells are empty, thus I thought "Type of building" should be the same, thus my implementation.

If I understand you correctly we should make both "non-mandatory" and not check the content? Excel will still provide the dropdown where the users can set one of the pre-specified values - or nothing.

Just let me know what you prefer and I'll quickly adjust it - easy fix :).

**Comment by gabroko** [comment]
Yes, Ventilation type should stay mandatory, but Type of building is not mandatory, but if filled out it should adhere to the standard nomeclature as defined in Definitions shhet... Thanks :-)

**Comment by retostauffer** [comment]
Done; data 'validation' is done by Excel (dropdown menu with data validation), `annex_validate()` will not check the content and allows them to be empty (non-mandatory).
